pkg/talos/networkconfig.go

```diff
@@ -93,6 +98,143 @@ func GenerateNetworkConfigBytes(ifCfg *config.IngressFirewall) ([]byte, error) {
 	return CombineYamlBytes(result), nil
 }
 
+func GenerateLinkAliasConfigBytes(devices []*v1alpha1.Device) ([]byte, error) {
+	var result [][]byte
+
+	for _, device := range devices {
+		if device.DeviceSelector == nil {
+			continue
+		}
+
+		aliasConfig := GenerateLinkAliasConfig(device)
+		if aliasConfig == nil {
+			continue
+		}
+
+		aliasBytes, err := marshalYaml(aliasConfig)
+		if err != nil {
+			return nil, err
+		}
+
+		result = append(result, aliasBytes)
+	}
+
+	if len(result) == 0 {
+		return nil, nil
+	}
+
+	return CombineYamlBytes(result), nil
+}
+
+func GenerateLinkAliasConfig(device *v1alpha1.Device) *network.LinkAliasConfigV1Alpha1 {
+	if device == nil || device.DeviceSelector == nil {
+		return nil
+	}
+
+	aliasConfig := network.NewLinkAliasConfigV1Alpha1(device.DeviceInterface)
+
+	celExpr, err := buildDeviceSelectorCELExpression(device.DeviceSelector)
+	if err != nil || celExpr == "" {
+		return nil
+	}
+
+	if err := aliasConfig.Selector.Match.UnmarshalText([]byte(celExpr)); err != nil {
+		return nil
+	}
+
+	return aliasConfig
+}
+
+func GenerateBondMemberAliasConfigBytes(devices []*v1alpha1.Device) ([]byte, error) {
+	var result [][]byte
+
+	for _, device := range devices {
+		if device.DeviceBond == nil || len(device.DeviceBond.BondDeviceSelectors) == 0 {
+			continue
+		}
+
+		for i, selector := range device.DeviceBond.BondDeviceSelectors {
+			aliasName := fmt.Sprintf("%s-m%d", device.DeviceInterface, i)
```

Comment:
Same as above, `device.DeviceInterface` is empty string here and we should give it a name manually, maybe `bond[0-9]`?